### PR TITLE
Fix conversation not populate sent message to view

### PR DESCRIPTION
**Pull request by yifanplanet**
# Code changes

- [x] Fixed conversation not populate sent message to view

# Readiness checklist

- [x] Cypress tests passing?
- [x] New cypress tests added?

# License

I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by pooja169usp — APPROVED** [review]
lgtm. Could we maybe add a test for this?

**Comment by yifanplanet** [comment]
> ould we maybe add a test for this?

Got it!

**Comment by yifanplanet** [comment]
> lgtm. Could we maybe add a test for this?

Thanks @pooja169usp, tests added!

File: components/conversation/src/init.spec.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-describe("Conversation Loading", () => {
+xdescribe("Conversation Loading", () => {
```
Comment (krisgardiner) [inline components/conversation/src/init.spec.js:1]: When will these get re-enabled? If there's no good reason to disable maybe we should try and fix the failing tests or remove them entirely if the coverage is not useful.

**Review by krisgardiner — APPROVED** [review]

**Pull request merged**